frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1093,904 +1093,931 @@ function ReaderFooter:addToMainMenu(menu_items)
-                            return self.settings.item_prefix == "icons"
+                            return self.settings.progress_pct_format == "0"
                         end,
                         callback = function()
-                            self.settings.item_prefix = "icons"
+                            self.settings.progress_pct_format = "0"
                             self:refreshFooter(true)
                         end,
                     },
                     {
                         text_func = function()
-                            local sym_tbl = {}
-                            for _, letter in pairs(symbol_prefix.letters) do
-                                table.insert(sym_tbl, letter)
-                            end
-                            return T(C_("Status bar", "Letters (%1)"), table.concat(sym_tbl, " "))
+                            return T(_("1 decimal place: %1"), self:progressPercentage(1))
                         end,
                         checked_func = function()
-                            return self.settings.item_prefix == "letters"
+                            return self.settings.progress_pct_format == "1"
                         end,
                         callback = function()
-                            self.settings.item_prefix = "letters"
+                            self.settings.progress_pct_format = "1"
                             self:refreshFooter(true)
                         end,
                     },
                     {
                         text_func = function()
-                            local sym_tbl = {}
-                            for _, letter in pairs(symbol_prefix.compact_items) do
-                                table.insert(sym_tbl, letter)
-                            end
-                            return T(C_("Status bar", "Compact (%1)"), table.concat(sym_tbl, " "))
+                            return T(_("2 decimal places: %1"), self:progressPercentage(2))
                         end,
                         checked_func = function()
-                            return self.settings.item_prefix == "compact_items"
+                            return self.settings.progress_pct_format == "2"
                         end,
                         callback = function()
-                            self.settings.item_prefix = "compact_items"
+                            self.settings.progress_pct_format = "2"
                             self:refreshFooter(true)
                         end,
                     },
                 },
             },
             {
-                text_func = function()
-                    local separator = self:get_separator_symbol()
-                    separator = separator ~= "" and separator or "none"
-                    return T(_("Item separator: %1"), separator)
-                end,
+                text = _("Max percentage of screen width used for text items"),
```

Comment:
This is too long. Probably not a new problem?

Reply:
I reduced it to: `Max pct. of screen width used for text items`

Reply:
Should be fine.